src/app/file-manager/login/login.component.html

```diff
@@ -1 +1,19 @@
-<!--TODO: Implement Ui-->
+<div class="flex-container center">
+    <p-toolbar styleClass="toolbar-accent toolbar-login">
+        <div class="logo">
+            <span class="bold">FC</span>Explorer
+        </div>
+        <p class="aclaration">Login using your Google account associated with <span class="medium">FireCloud</span></p>
+        <div class="flex-container center">
+            <button mat-raised-button class="button-primary" (click)="showError()">Sign In</button>
+        </div>
+    </p-toolbar>
+</div>
+
+<div class="flex-container center">
+    <p-messages class="error-messages" [(value)]="msgs"></p-messages>
+</div>
+
+<div class="flex-container center">
+    <p class="login-message">Not a FireCloud user yet? Please go to: <span class="bold">https://portal.firecloud.org/</span> to create a user account.</p>
```

Comment:
This should use `environment.FIRECLOUD_API` instead of hardcoding the firecloud location.

Reply:
The new FIRECLOUD_API value will be https://api.firecloud.org so she won't be able to use it for that purpose. We can create a new variable with the URL portal.firecloud.org on environment.ts

Reply:
Agree with you, but right now external links are not working from electron.